src/layer/__tests__/legacy.ts

```diff
@@ -0,0 +1,67 @@
+import 'jest-setup';
+import moment from 'moment';
+
+import { parseLegacyWmsGetMapParams, BBox, CRS_EPSG3857, GetMapParams, PreviewMode } from 'src';
+
+test('parseLegacyWmsGetMapParams with evalscripturl', () => {
+  const evalscriptUrlOriginal =
+    'https://gist.githubusercontent.com/sinergise-anze/33fe78d9b1fd24d656882d7916a83d4d/raw/295b9d9f033c7e3f1e533363322d84846808564c/data-fusion-evalscript.js';
+  const wmsParams = {
```

Comment:
Minor: I would also add `gain` and `gamma` parameters as well.

Reply:
Added.

Reply:
Nice, resolving.